lib/modules/manager/mix/extract.ts

```diff
@@ -44,6 +45,9 @@ export async function extractPackageFile(
         const ref = refRegexp.exec(opts)?.groups?.value;
         const branchOrTag = branchOrTagRegexp.exec(opts)?.groups?.value;
         const organization = organizationRegexp.exec(opts)?.groups?.value;
+        const hex =
+          hexRegexp.exec(opts)?.groups?.strValue ??
+          hexRegexp.exec(opts)?.groups?.atomValue;
```

Comment:
Both of the tests added seem like the same syntax to me. What's strValue vs atomValue? Is there an example of that?

Reply:
I did the example [here](https://github.com/bernardo-martinez/renovate-reproduction-32834/blob/main/mix.exs#L19). As you can see the option accepts either a string or an atom (atoms in elixir being _interned strings_)